recipe/meta.yaml

```diff
@@ -30,10 +30,10 @@ test:
 
 about:
   home: https://chemfiles.org
-  license: MPL-v2.0
-  license_file: LICENCE.txt
+  license: BSD
```

Comment:
Is this `2-Clause` or `3-Clause`?

Reply:
3-clause. I tried to look at how one should specify licenses, but I could not find any definitive information. Should I put it as `BSD-3-Clauses` ?

Reply:
Typically we do `BSD 3-Clause`, but an extra `-` is probably fine. We aren't too picky about it yet.

Reply:
The extra dash comes from https://spdx.org/licenses/ identifiers, which is what cargo (rust package manager) uses.